### PR TITLE
Replace TPR with CRD

### PR DESCRIPTION
**Pull request by ikitiki**
ref https://github.com/zalando-incubator/postgres-operator/issues/98

### CODE REVIEW

**Comment by Jan-M** [comment]
Do the example manifests need an update?

**Comment by ikitiki** [comment]
actually example manifest has been changed:
https://github.com/zalando-incubator/postgres-operator/pull/117/files#diff-c2c42f271df26541b4bea761a08f6c3dL2

**Comment by Jan-M** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

File: pkg/controller/util.go [hunk 1]
```diff
@@ -28,36 +29,59 @@ func (c *Controller) makeClusterConfig() cluster.Config {
...
 func (c *Controller) clusterWorkerID(clusterName spec.NamespacedName) uint32 {
 	return crc32.ChecksumIEEE([]byte(clusterName.String())) % c.opConfig.Workers
 }
 
-func (c *Controller) createTPR() error {
-	tpr := thirdPartyResource(constants.TPRName)
+func (c *Controller) createCRD() error {
+	crd := &apiextv1beta1.CustomResourceDefinition{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: constants.CRDResource + "." + constants.CRDGroup,
```
Comment (alexeyklyukin) [inline pkg/controller/util.go]: Do we need to specify the namespace here as well (since we are using the NamespacedScope)?

Comment (ikitiki) [reply pkg/controller/util.go]: makes sense

Comment (ikitiki) [reply pkg/controller/util.go]: actually no, CRD is cluster wide

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**